docs/retry-activities-and-workflows.md

```diff
@@ -0,0 +1,185 @@
+---
+id: retry-activities-and-workflows
+title: How to retry Activities and Workflows
+sidebar_label: Retry Activities and Workflows
+---
+
+Temporal is built to expect failures from Activities and Workflows. Retrying Activity or Workflow executions is part of what makes the Temporal system fault tolerant.
+
+In this guide we cover the difference between Activity and Workflow retries and how to use them.
+
+## Activity retries
+
+The nature of an [Activity](docs/activities) is to encapsulate logic that is non-deterministic. With Temporal, the paradigm is to not explicitly handle errors and retries within the Activity logic. Instead, the Activity should return the error directly to the Workflow which will trigger a retry per the Activity's retry policy. Each Activity can have its own retry policy, can use a shared one, or not specify one at all and use Temporal's default retry policy. Said another way, if `RetryOptions` are not specified within the `ActivityOptions` then the Activity will be retried per Temporal's default configuration.
```

Comment:
```suggestion
[Activities](docs/activities) are intended to encapsulate logic that is non-deterministic. With Temporal, the paradigm is to not explicitly handle errors and retries within the Activity logic. Instead, the Activity should return the error directly to the Workflow which will trigger a retry per the Activity's retry policy. An Activity can either specify:

* its own retry policy
* a shared one
* nothing, which uses the default retry policy. 

Said another way, if `RetryOptions` are not specified within the `ActivityOptions` then the Activity will be retried per Temporal's default configuration.
```

Reply:
I think the second and third sentences could be reworded to be way easier to read.

Also maybe add the default RetryOptions